iron-spi-aws-inttest/pom.xml

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>iron</artifactId>
+        <groupId>io.axway.iron</groupId>
+        <version>${revision}</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>iron-spi-aws-inttest</artifactId>
+    <name>${project.artifactId}</name>
+    <description>Iron transaction and snapshot store SPI AWS implementations Inttest</description>
+
+    <properties>
+        <bnd.Import-Package/>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>io.axway.iron</groupId>
+            <artifactId>iron-spi</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>io.axway.iron</groupId>
+            <artifactId>iron-spi-aws</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>io.axway.iron</groupId>
+            <artifactId>iron-core</artifactId>
+            <scope>compile</scope>
+        </dependency>
+        <dependency>
+            <groupId>io.axway.iron</groupId>
+            <artifactId>iron-spi-test</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>io.axway.iron</groupId>
+            <artifactId>iron-spi-jackson</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.testng</groupId>
+            <artifactId>testng</artifactId>
+            <scope>compile</scope>
+        </dependency>
+    </dependencies>
+
+    <profiles>
+        <profile>
+            <id>localstack</id>
+            <build>
+                <resources>
+                    <resource>
+                        <directory>
+                            src/test/localstack-resources
```

Comment:
why not using the standard "resources" directory ?

Reply:
Fixed.

Reply:
so why redefine it since it's the default resources location ?